elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/ElasticsearchFlowStage.scala

```diff
@@ -57,7 +57,7 @@ object IncomingUpsertMessage {
 
   // Java-api - without passThrough
   def create[T](id: String, source: T): IncomingMessage[T, NotUsed] =
-    IncomingUpdateMessage(id, source)
+    IncomingUpsertMessage(id, source)
```

Comment:
Ahh. Nice catch. :)

Reply:
I should have written also Java test from the beginning...